test/verifying_paymaster.test.ts

```diff
@@ -77,15 +87,50 @@ describe('EntryPoint with VerifyingPaymaster', function () {
 
     it('succeed with valid signature', async () => {
       const userOp1 = await fillAndSign({
-        sender: account.address
+        sender: account.address,
+        paymasterAndData: hexConcat([paymaster.address, '0x000000007B9F2E9D', '0x0000000000000000'])
       }, accountOwner, entryPoint)
       const hash = await paymaster.getHash(userOp1)
```

Comment:
here we could have  `getHash(userOp, since, until)` (no need to pass the nonce - this view method can read it from the contract)